exercises/acronym/example.go

```diff
@@ -0,0 +1,21 @@
+package acronym
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+const testVersion = 1
+
+func abbreviate(s string) (string, error) {
+	regex, _ := regexp.Compile("[A-Z]+[a-z]*|[a-z]+")
+	wordList := regex.FindAllString(s, -1)
+	abbr := []string{}
+
+	for _, word := range wordList {
```

Comment:
Instead of `wordList`, let's use `words`. Then the loop reads _for singular in range of plural_.